sermant-plugins/sermant-router/spring-router-plugin/src/main/java/io/sermant/router/spring/handler/AbstractHandler.java

```diff
@@ -87,4 +100,35 @@ protected void collectLaneCountMetric(Map<String, List<String>> laneTag) {
             });
         });
     }
+
+    /**
+     * Transparent transmission marks the key entity
+     *
+     * @author provenceee
+     * @since 2023-02-21
+     */
+    public static class Keys {
```

Comment:
no need define it as inner class,for Keys is used in other classes

Reply:
ok